cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -54,6 +54,14 @@
 PCLUSTER_LAMBDA_PREFIX = "pcluster-"
 
 
+def _get_resource_combination_name(*resource_names, partial_length=7, hash_length=16):
```

Comment:
NIT: We should remove the `_` since this is a public function, the underscore(`_`) is normally associated with a private function by convention. A more approriate name could also be `generate_fixed_size_resource_name`